src/scripts.js

```diff
@@ -0,0 +1,67 @@
+'use strict';
+
+let initialState = {
+    clothers: [
+      'Apron',
+      'Belt',
+      'Cardigan',
+      'Dress',
+      'Earrings',
+      'Fur',
+      'Coat',
+      'Gloves',
+      'Hat'
+    ],
+
+    selectedIndex: -1,
+    inEdit: false
+};
+
+function getNextState(state = initialState, action) {
+  let newState;
+  switch (action.type) {
+      case SELECT:
+          newState = {
```

Comment:
will be better to return new state here, and you do not need break and additional variable